src/BatchBuilder.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Dew\Tablestore;
+
+use Dew\Tablestore\Concerns\HasConditions;
+use Protos\OperationType;
+
+class BatchBuilder
+{
+    use HasConditions;
+
+    /**
+     * The operation type.
+     */
+    public ?int $operation = null;
+
+    /**
+     * The row writer.
+     */
+    public ?RowWriter $row = null;
+
+    /**
+     * Create a batch builder.
+     */
+    public function __construct(
+        protected string $table
+    ) {
+        //
+    }
+
+    /**
+     * Filter rows by the given primary keys.
+     *
+     * @param  (\Dew\Tablestore\Cells\Cell&\Dew\Tablestore\Contracts\PrimaryKey)[]  $primaryKeys
+     */
+    public function where(array $primaryKeys): self
+    {
+        $this->wheres = $primaryKeys;
+
+        return $this;
+    }
```

Comment:
This duplicated `where` method is already defined in `HasConditions` trait which could be removed.